--- ../../../binutils-2.20/gas/configure.tgt	2009-09-09 04:13:28.000000000 -0400
+++ gas/configure.tgt	2010-04-03 19:13:26.000000000 -0400
@@ -236,6 +236,7 @@ case ${generic_target} in
   i386-*-chaos)				fmt=elf ;;
   i386-*-rdos*)				fmt=elf ;;
   i386-*-darwin*)			fmt=macho ;;
+  i386-*-xomb*)				fmt=elf ;;
 
   i860-*-*)				fmt=elf endian=little ;;
 
--- ../../../binutils-2.20/gas/configure.tgt	2009-09-09 04:13:28.000000000 -0400
+++ gas/configure.tgt	2010-04-03 19:13:26.000000000 -0400
@@ -236,6 +236,7 @@ case ${generic_target} in
   i386-*-chaos)				fmt=elf ;;
   i386-*-rdos*)				fmt=elf ;;
   i386-*-darwin*)			fmt=macho ;;
+  i386-*-xomb*)				fmt=elf ;;
 
   i860-*-*)				fmt=elf endian=little ;;
 
